Sources/Quick/NSString+QCKSelectorName.m

```diff
@@ -27,7 +27,11 @@ - (NSString *)qck_selectorName {
 
     NSArray *validComponents = [self componentsSeparatedByCharactersInSet:invalidCharacters];
 
-    return [validComponents componentsJoinedByString:@"_"];
+    NSString *result = [validComponents componentsJoinedByString:@"_"];
+    
+    return ([result length] == 0
+            ? @"_"
+            : result);
```

Comment:
Interesting... I guess without this additional code, `it("")` would have a selector name of `""`, is that right? What sort of behavior was that causing? Would `it("")` examples never be run?